icu4c/source/common/brkiter.cpp

```diff
@@ -437,6 +437,14 @@ BreakIterator::makeInstance(const Locale& loc, int32_t kind, UErrorCode& status)
                 uprv_strcat(lbType, "_");
                 uprv_strcat(lbType, lbKeyValue);
             }
+            if (strcmp(lbType, "line") == 0) {
+                char lwKeyValue[kKeyValueLenMax] = { 0 };
```

Comment:
CharString lwKeyValue;
CharStringByteSink valueSink(& lwKeyValue);
loc.getKeywordValue("lw", valueSink, kvStatus);
if (U_SUCCESS(kvStatus) && lwKeyValue == "phrase") {
   uprv_strcat(lbType, "_");
    uprv_strcat(lbType, lwKeyValue.data());
}

see similar usage in line 1732 of source/common/locid.cpp